### DISCUSSION TITLE
问个细节问题

### DISCUSSION THREAD

**Discussion by PureDimension** [liusy58/NJU_AdvancedProgramming_SP22#145] [Q&A] [answered]
卖家模式下查看历史订单是不是只能看到自己卖的订单，而买家模式下查看历史订单只能看到自己买的订单
还是两者是通用的hmm

**Comment by PureDimension** [comment, marked as answer]
还有是不是被封禁的用户就无法登录自己的账号了

**Reply by binbin00** [reply]
1. ppt里给出的样例的意思显然是，卖家只能看到卖的，买家只能看到买的，当然你都展示也是可以的。
2. 是的。
